rfcs/top-navigation-silhouette.md

```diff
@@ -0,0 +1,59 @@
+# Top Navigation Silhouette
+
+This silhouette provides proper styles for wrapper element to be used as Top Navigation. By default element has no visual styles.
+Silhouettes comply with Base UI grid layout, so they are responsive to resizing and stretching, changing sizes, fonts, etc
+
+## Usage
+
+### Basic usage
+
+```javascript
+import * as React from 'react';
+import {TopNavigation} from 'baseui/silhouette/top-navigation';
+
+export default () => <TopNavigation><button>Menu</button><span>Uber</span></TopNavigation>;
```

Comment:
can you break this into multiple lines? 
if you are using the button as an example, can you use the Button component?

Reply:
ok